docs/configuration.md

```diff
@@ -0,0 +1,457 @@
+>>Value must be in the range of 0-1.
+>> #### Example:
+>>```js
+>>var config = {
+>>    playback: {
+>>        volume: 0.5
+>>    }
+>>};
+>>```
+>## 
+>>### config.playback.playsinline
+>>##### Type: `boolean`
+>>##### Default: `true`
+>>##### Description: A Boolean attribute which indicates that the video is to be played "inline", that is within the element's playback area.
+>>Especially relevant for playing on iPhone devices, where if the value is set to false, the video will be played using the AV Player (iOS native video player).
+>## 
+>>### config.playback.preload
+>>##### Type: `string`
+>>##### Default: `"none"`
+>>##### Description: Indicates whether the video should be preloaded or not.
+>>Possible values:
+>> - `"none"`: indicates that the video should not be preloaded.
+>> - `"auto"`: indicates that the whole video file could be downloaded, even if the user is not expected to use it.
+>##
+>>### config.playback.autoplay/allowMutedAutoPlay
+>>for `autoplay` & `allowMutedAutoPlay` options read [here](autoplay.md).
+>## 
+>>### config.playback.muted
+>>##### Type: `boolean`
+>>##### Default: `false`
+>>##### Description: Indicates whether the video should be muted or not.
+>>A Boolean attribute which indicates the default setting of the audio contained in the video. If set, the audio will be initially silenced. Its default value is false, meaning that the audio will be played when the video is played.
+>##
+>>### config.playback.options
+>>##### Type: `PKPlaybackOptionsObject`
+>>```js
+>>{
+>>  html5: {
+>>    hls: Object,
+>>    dash: Object
+>>  }
+>>}
+>>```
+>>##### Default: 
+>>```js
+>>{
+>>  html5: {
+>>    hls: {},
+>>    dash: {}
+>>  }
+>>}
+>>```
+>>##### Description: Defined the media source adapters configurations.
+>>For `hls` configuration, see [hls.js](https://github.com/video-dev/hls.js/blob/master/doc/API.md#fine-tuning) documentation.
+>For `dash` configuration, see [shaka-player](https://shaka-player-demo.appspot.com/docs/api/tutorial-config.html) documentation.
+>## 
+>>### config.playback.preferNative
+>>##### Type: `PKPreferNativeConfig`
+>>```js
+>>{
+>>  hls: boolean,
+>>  dash: boolean
+>>}
+>>```
+>>##### Default: 
+>>```js
+>>{
+>>  hls: false,
+>>  dash: false
+>>}
+>>```
+>>##### Description: Whether to prefer native browser playback (if supported) with media source extensions.
+>>If one of the values sets to true and the player chooses to play the truthly media source extension, the player will try to play it natively if supported by the browser. 
+>> #### Example:
+>>Lets assume the following configuration:
+>>```js
+>>var config = {
+>>  playback: {
+>>    preferNative:  {
+>>      hls: true
+>>    }
+>>  }
+>>};
+>>```
+>> In case we're running in _Safari_ browser, the player will play native hls playback managed by the _Safari_ browser. But in case we're running in browser which hls playback is not supported natively, _Chrome_ for instance, the player will play hls using the `hls.js` library.
```

Comment:
If the player is running on a _Safari_  browser, the player will use the native hls playback managed by the _Safari_  browser. However, if running on a browser in which hls playback is not supported natively, for example, _Chrome_, the player will play hls using the 'hls.js' library.